datajoint/external.py

```diff
@@ -326,7 +329,9 @@ def _need_checksum(local_filepath):
                             f"'{local_filepath}' downloaded but did not pass checksum'"
                         )
             if not _need_checksum(local_filepath):
-                print(f"WARNING SKIPPED CHECKSUM FOR FILE WITH HASH: {contents_hash}")
+                logger.warning(
+                    f"WARNING SKIPPED CHECKSUM FOR FILE WITH HASH: {contents_hash}"
```

Comment:
why all caps?

Reply:
No reason, do you think it should be normal case?